### PR TITLE
Add button to delete all comments

### PR DESCRIPTION
**Pull request by TsarN**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    Transaction txn = datastore.beginTransaction();
+
+    try {
+        for (Entity entity : results.asIterable()) {
+            datastore.delete(txn, entity.getKey());
+        }
+        txn.commit();
+    } finally {
+        if (txn.isActive()) {
+            txn.rollback();
```
Comment (DarthGandalf) [inline portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java:54]: Should it try again?

File: portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import com.google.appengine.api.datastore.Transaction;
+
+/** Servlet that returns comments. */
+@WebServlet("/commentDeleteAll")
+public class CommentDeleteAllServlet extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    
+    // Queries aren't allowed in transactions (or maybe I'm doing something wrong)
```
Comment (DarthGandalf) [inline portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java:40]: That sounds strange

File: portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java (hunk 1 above)
Comment (TsarN) [reply portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java:54]: I think it's recommended to try again with exponential backoff. Doesn't seem applicable in the context of HTTP request though and having some sort of a task queue is probably too much for this task.

File: portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java (hunk 2 above)
Comment (TsarN) [reply portfolio/src/main/java/com/google/sps/servlets/CommentDeleteAllServlet.java]: How about now?

**Pull request merged**